resources/views/ceklulus.blade.php

```diff
@@ -1,160 +1,158 @@
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ asset('/assets/favicon/favicon-16x16.png') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/navbar-ex.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/footer.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/login-caas-menu.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/about.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/home.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/form-style.css') }}">
+  <link rel="stylesheet" type="text/css" href="{{ asset('/css/main.css') }}">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"></script>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Electrolize&family=Fira+Code:wght@400&display=swap" rel="stylesheet">
+
 </head>
-<body>
 
-@include('layouts/navbarCaasLogout')
-@section('navcaaslogout')
-@endsection
+<body style="background-color: #2c4d16;">
 
-<section id="main-nim">
-  @if($Active==1 && $urut_tahap==$current_tahap)
-    <div class="container p-5 pt-0">
-      <div class="d-flex justify-content-center pb-4">
-        <div>
-          <img src="{{asset('/assets/dlor-blue.png')}}" alt="logo" class="dlor-logo-announce">
-        </div>
-      </div>
+  @include('layouts/navbarCaasLogout')
+  @section('navcaaslogout')
+  @endsection
+
+  <section id="landing-section">
+    @if($Active===1 && $urut_tahap===$current_tahap)
+    <div class="container">
       <div class="d-flex justify-content-center">
-        <div style="background-color: #FFC844" class="checker-box pb-5">
+        <div class="checker-box">
           <div class="text-center text-nim-head">
-            <span>Apakah kamu lulus Tahap {{ $namatahap }}?</span>  
+            <span>Apakah kamu lulus Tahap {{ $namatahap }}?</span>
           </div>
           <div class="text-center pt-3 pb-3">
             <div>
-              <span style="font-weight: 600" class="uppercase Welcome-text">
-              {{$nama}}
+              <span class="uppercase text-nim">
+                {{$nama}}
+              </span>
+            </div>
+            <div>
+              <span class="text-nim">
+                NIM : {{$nim}}
+              </span>
+            </div>
+            @if($isLolos==1 && $urut_tahap==$current_tahap)
+            <div class="pt-3">
+              <span style="font-size:25px;font-weight: 700;color:green;" class="Welcome-text">
+                {!!nl2br(e($lulustext))!!}
+              </span>
```

Comment:
```suggestion
              <span style="font-size:25px;font-weight: 700;color:whitesmoke;background-color:green;border-radius:15px;" class="Welcome-text">
                {!!nl2br(e($lulustext))!!}
              </span>
```
nah coba ganti gitu biar text nya gak samar, tapi disesuain lagi aja radius sama kawan2nya dip

Reply:
Oh iya saya font-size nya digedein lagi dip, itu di desktop kyknya kekecilan ya?

Reply:
Kalo gini gimana bang?
```suggestion
              <span style="font-size:2.5vw;font-weight: 700;color:whitesmoke;background-color:green;border-radius:17px; padding:10px 15px 10px 15px" class="Welcome-text">
                {!!nl2br(e($lulustext))!!}
              </span>
```
![image](https://user-images.githubusercontent.com/88444330/160155008-0596c35e-d7b7-49bb-af74-d9482360419b.png)

Reply:
oke bolehh, nanti next PR disesuain lagi aja warna nya biar lebih bagus, tapi itu udah bener